backend/app/conversations/callConvo.js

```diff
@@ -217,6 +217,139 @@ function howDidItGoConvo(user, message) {
   )
 }
 
+function noNextRepResponse(user, message, numCalls) {
+  return botReply(user, {
+    attachment: {
+      type: 'image',
+      payload: {
+        url: 'https://storage.googleapis.com/callparty/success.gif'
+      }
+    }
+  })
+  .then(() => {
+    // if the user is the first caller
+    if (numCalls <= 1) {
+      return botReply(user, stripIndent`
+        Congrats, you’re the first caller on this issue! You’ve joined the ranks of other famous firsts in American History. We'll reach out when we have updates and an outcome on the issue.
+      `)
+    }
+    // if the user is only person who has made calls, then it's weird to tell them how many calls so far so remove that part
+    else if (numCalls <= user.convoData.currentRepresentativeIndex + 1) {
```

Comment:
I think this condition passes the situation you described (i don't think calls remaining effects curRepIndex)

I'll add a more clear check. Altho as I wrote this I came to the feeling that this was already overkill. I think we should both be mindful to not let our perfectionism get in the way of overall progress (I'm guilty of this 100% by even writing this clause). We should probably try to structure conversations so that we don't have to worry about these little details (@kelseyah), that way we can focus on stuff more relevant to the core functionality